ops/terraform/env/mgmt/base_config/values/mgmt.yaml

```diff
@@ -15,6 +15,7 @@ bfd:
+        <<CIPHER>>AgV4oB_GrEl4Sd94CLbzrkAG4ILgzk1Q7uI6vb3StRElN4AAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREF2UFc4aFJabzVuMTQvTGhNTUVQYnhCRHlTWVRTNmEvaDIybEFRazJaNTU0cEJHRzVKYXVQM0VQQ2U4RVlqY2RCUT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwGeIocpXdrdD5Pe4tobj-zdAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMLT2X8leS08vJAhP5AgEQgDtfvfyxla1GPu-VSHfegwP1F6WmKpqz23IiVXFb9tl7MsS7rqN1jagwWbzcuPNSuMLN3WXM4rYCkD4xgwIAABAAol_Onud1ACfFBN-_lmN4qVzsDjSM_SWnHr2yuGDd1VkKEbwvY5jGovVfhZT5-N4Y_____wAAAAEAAAAAAAAAAAAAAAEAAAAE79fz_9Z6mcfnGvVtcs-d2t89w3cAZzBlAjBThHAuGkABusrYJvhcZLjAVIu990apXxICT-aLlK36BRUVKIR1BLIKGtLaPzdHensCMQCeTMdqBtR1FjyDWTF0wduw34QuejUnTjTAUGDIJ8_2oPMsamN2BffSuMHE3KNFr40=<</CIPHER>>: <<CIPHER>>AgV4YhUyTdWN2v59XdkkRHNY3WWPaBT3Xz1PByBoue1ii-0AdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFrU0pXbkNPRnVYcWtyRG11SmxPN1hldjJwNzFUb2lBOVd5QW5lZ0t0eFdJaGU1OW40SjIveW4yQkpMWkdoWm1HUT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwEvfwsELvqz3W9CdPZIzISjAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMSMH8FdqfrGkQRpuLAgEQgDuuGzMLWnm4fCnPNvy23VmuOuN9H13I6IThLxPJSaiZS1j7m5gzBNnXOCueQLe2arBfTeyYePtoE9bKIgIAABAA-kJWj0AO0n1SjiKqxEAvvVDK48ctQ_kxHGDeBUaZpFUPUJMYAdBOn9iM54eOhhPt_____wAAAAEAAAAAAAAAAAAAAAEAAAOiRQsUeuvul2bIA_FnSHWHxiw-gzt0VH31ZZBsYfjvAA1JPKqlIFJyNaOuLhUsFVLeWA8kDzBA3z4wAXY0vJ1fsMg8p0aMVZQY4cjTwKvU06uxtXC6mn_qaK-or8P8_yANqp132nSaLd25eVTc7kzVNzKrzerCb5AQwuaqUjyZs7NNReMd0GhIy0V_cSkiN2NXDTrigit3Bc1IbHCrvyaX24aDY7wLunfdxa6QtVgPRvdWwFotqQC67s4TgfhO8kydA-cV1UxPGV7R29J1W_ghYbxGigqhy7czFpOxNcdkRz9fWrXvu4YRZuzLE_83b9-F80-zuTpuh3Vh6ARjBKCaxZJRlRHf3MY_2VgBAjOwl3GHqnBkr10qYDpUpX6MZMk8h999pUNRFm3HneIiRDIqnBDo9uQCxVTRLpqSxGqC3yivppnQikHXoaV2oa5k6ahNVcBfMPnB8KJew2pbechP7MuzdKxS35Z-uazq50MhdkFfC1RHbXD-puycmnA22gWpf6UzzDC4AYnvbMAKirQrcoy_BLe2rerGUfV_s0PbEgoKqgnXjITLByPaZ3IpzYABGKM_T644VRl_ndyaYnP1VTn0PQ6ylDqpxEyMvElQYaJ83rYT-XhjEmAdWzroYDW0NwK9cHIfflKGcmvknhDjaNarurnVaOUVqkHzt77Qc6Y-aQebGSbGhxyI6sdVKm2K3rx5LOLb5qdP6wq3_sDX0lme4NDgGT_TgIWqyKp_iz1dOLXV3O11jlgrPSvXll8R9I3P2-YM3xVszt6TZMx7WOMYn3ATP-tyESFVknT1S0ulI1QwGXgjTczbyiYUh7RLOz11NFkNwrzFtqfkSorHCeG4JA2hC8pKVDa2K2tbFbHE5i8iYl4jHF_7wX4acCD1N2yj3Jxsz2sScTJ0lAhh_XH9I8m6JZ0vrXJVYL4PAr8k4PncGiQB7k-UUUqUDyuWinl5O1YIr6cujPlYa65_SCygGTbk7OBM4kwCELOk-HZY1aX0OU8xpczjOOOH6QqUCD5MStF4drRvcdFsowKjhFc_gxSpdkFZ2WzHtvGpnQB9341vc9TT40j7tLyRIsxOlSdHAJZiZqPOIH8tQQfPiUWnMfYEXsE5c90IUofjdiTWXgvIRVg-Fs2u8xxh_mEn4RPM4hyGVSPa6Uhiz4u5JxHmPky-QivmGT9jrz8eEfcmmV7wsVNYg3egLdDETjxf0yxQrKWD9BfmxI3JLap8lEFjSBXN5MMXoVCSNrDJQPumhQBnMGUCMEYDfPmzpFRTflz7GKg19lYHT8OTUQSHv4neYlm0xWiC_k-JucyjIq0mPOCD9sGnKAIxANRjqfMqiAcJ0kxvZp__jXykp4g3ZRM-9mrxRxPMz1EMnJBMdqB0fyR5JgY_W8xJXw==<</CIPHER>>
```

Comment:
daniellangrill@navapbc.com is in there twice, for "identity_type": "QUICKSIGHT". Can probably consolidate this entry.

Reply:
This is fixed. Please verify. Thanks.